patching/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -165,6 +165,8 @@ async function diffFolders (folderA: string, folderB: string): Promise<string> {
     .replace(new RegExp(escapeStringRegexp(`${folderAN}/`), 'g'), '')
     .replace(new RegExp(escapeStringRegexp(`${folderBN}/`), 'g'), '')
     .replace(/\n\\ No newline at end of file\n$/, '\n')
+    .replace(/^diff --git a\/.*\.DS_Store b\/.*\.DS_Store[\s\S]*?(?=^diff --git)/gm, '')
```

Comment:
The cause was that the --no-index option caused .gitignore to be ignored.
However, the --no-index option cannot be turned off because the change was made within node_modules.
Also, I tried to exclude certain files with the git diff command, but that didn't work either, so I fixed it with a regular expression

```js
await execa('git', ['-c', 'core.safecrlf=false', 'diff', '--src-prefix=a/', '--dst-prefix=b/', '--ignore-cr-at-eol', '--irreversible-delete', '--full-index', '--no-index', '--text', '--no-ext-diff', folderAN, folderBN]
```

This regex corresponds to the difference except for the tail in PATCH.